src/components/route-search/CategorySection.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import Chip from '../chip/Chip';
+import styled from '@emotion/styled';
+
+function CategorySection() {
+  return (
+    <Wrapper>
+      {CATEGORY_DUMMY.map((name) => (
+        <Chip label={name} key={name} />
+      ))}
+    </Wrapper>
+  );
+}
+
+export default CategorySection;
+
+const CATEGORY_DUMMY = ['물놀이', '디프만', '호캉스', '일상', '운동', '유럽여행'];
+
+const Wrapper = styled.div`
+  overflow: auto;
+  white-space: nowrap;
+`;
```

Comment:
<img width="490" alt="스크린샷 2022-11-25 오후 2 50 04" src="https://user-images.githubusercontent.com/26461307/203910008-b8c98a38-5841-480a-9e0a-2afba62e3216.png">

`gap: 4px`이 있어야 할 거 같아요 !

---

추가적으로 이렇게 속성을 사용하는 것이 더 직관적이지 않을까... 하는 개인적인 생각이에요 !
```suggestion
const Wrapper = styled.div`
  overflow-x: scroll;
  white-space: nowrap;
`;
```

Reply:
> <img alt="스크린샷 2022-11-25 오후 2 50 04" width="490" src="https://user-images.githubusercontent.com/26461307/203910008-b8c98a38-5841-480a-9e0a-2afba62e3216.png">
> 
> `gap: 4px`이 있어야 할 거 같아요 !
> 
> 추가적으로 이렇게 속성을 사용하는 것이 더 직관적이지 않을까... 하는 개인적인 생각이에요 !

`overflow: scroll`은 컨텐츠가 넘치는지 여부와 관계없이 스크롤을 생성하고, `overflow: auto`는 넘칠 때에만 스크롤을 생성한다고 하더라구요! `overflow-x: auto`로 수정했습니다 👍🏻

Reply:
오호 그렇군요 !! ㅈ ㅓ도 배워갑니다 ~~ 👍 👍